src/Repository/ShippingExportRepository.php

```diff
@@ -16,18 +16,38 @@
 
 class ShippingExportRepository extends EntityRepository implements ShippingExportRepositoryInterface
 {
+    public const NEW_STATE_PARAMETER = 'newState';
+
+    public const PENDING_STATE_PARAMETER = 'pendingState';
+
     public function createListQueryBuilder(): QueryBuilder
     {
         return $this->createQueryBuilder('o')
             ->leftJoin('o.shipment', 'shipment')
         ;
     }
 
+    /**
```

Comment:
Does it have any value?

Reply:
It inherits `@depracated` from the interface.